docs/excel/excel-add-ins-pivottables.md

```diff
@@ -39,57 +39,58 @@ This PivotTable could be generated through the JavaScript API or through the Exc
 
-```typescript
-await Excel.run(async (context) => {
-    // creating a PivotTable named "Farm Sales" on the current worksheet at cell A22 with data from the range A1:E21
-    context.workbook.worksheets.getActiveWorksheet().pivotTables.add("Farm Sales", "A1:E21", "A22");
+```js
+Excel.run(function (context) {
+    // Create a PivotTable named "Farm Sales" on the current worksheet at cell
+    // A22 with data from the range A1:E21.
+    context.workbook.worksheets.getActiveWorksheet().pivotTables.add(
+      "Farm Sales", "A1:E21", "A22");
 
-    await context.sync();
+    return context.sync();
 });
 ```
 
 ### Create a PivotTable with Range objects
 
-```typescript
-await Excel.run(async (context) => {
-    // creating a PivotTable named "Farm Sales" on a worksheet called "PivotWorksheet" at cell A2
-    // the data comes from the worksheet "DataWorksheet" across the range A1:E21
-    const rangeToAnalyze = context.workbook.worksheets.getItem("DataWorksheet").getRange("A1:E21");
-    const rangeToPlacePivot = context.workbook.worksheets.getItem("PivotWorksheet").getRange("A2");
+```js
+Excel.run(function (context) {
+    // Create a PivotTable named "Farm Sales" on a worksheet called "PivotWorksheet" at cell A2
+    // the data comes from the worksheet "DataWorksheet" across the range A1:E21.
+    var rangeToAnalyze = context.workbook.worksheets.getItem("DataWorksheet").getRange("A1:E21");
+    var rangeToPlacePivot = context.workbook.worksheets.getItem("PivotWorksheet").getRange("A2");
     context.workbook.worksheets.getItem("PivotWorksheet").pivotTables.add(
-        "Farm Sales", rangeToAnalyze, rangeToPlacePivot);
+      "Farm Sales", rangeToAnalyze, rangeToPlacePivot);
 
-    await context.sync();
+    return context.sync();
 });
 ```
 
 ### Create a PivotTable at the workbook level
 
-```typescript
-await Excel.run(async (context) => {
-    // creating a PivotTable named "Farm Sales" on a worksheet called "PivotWorksheet" at cell A2
-    // the data is from the worksheet "DataWorksheet" across the range A1:E21
-    context.workbook.pivotTables.add("Farm Sales", "DataWorksheet!A1:E21", "PivotWorksheet!A2");
+```js
+Excel.run(function (context) {
+    // Create a PivotTable named "Farm Sales" on a worksheet called "PivotWorksheet" at cell A2
+    // the data is from the worksheet "DataWorksheet" across the range A1:E21.
+    context.workbook.pivotTables.add(
+        "Farm Sales", "DataWorksheet!A1:E21", "PivotWorksheet!A2");
 
-    await context.sync();
+    return context.sync();
 });
 ```
 
 ## Use an existing PivotTable
 
-Manually created PivotTables are also accessible through the PivotTable collection of the workbook or of individual worksheets. 
+Manually created PivotTables are also accessible through the PivotTable collection of the workbook or of individual worksheets. The following code gets the first PivotTable in the workbook. It then gives the table a name for easy future reference.
```

Comment:
Doesn't seem like this accurately describes the code sample.  The sample is just getting the PivotTable named "MyPivot".